crates/flashblocks-rpc/src/state.rs

```diff
@@ -378,12 +367,11 @@ where
         let earliest_block_number = flashblocks_per_block.keys().min().unwrap();
         let canonical_block = earliest_block_number - 1;
         let mut last_block_header = self.client.header_by_number(canonical_block)?.ok_or(eyre!(
-            "Failed to extract header for canonical block number {}. This is okay if your node is not fully synced to tip yet.",
+            "Failed to extract header for canonical block number {}. Allow 10 to 15 minutes after a restart/redeploy or after your node has synced to tip.",
```

Comment:
IMO the old message was better. The 10-15 minutes doesn't feel accurate if you're restoring from a snapshot/or you've been offline for a while.